--- cloog-0.21.1.orig/Makefile.am	Sat Sep 20 23:06:28 2025
+++ cloog-0.21.1/Makefile.am	Sat Sep 20 23:16:41 2025
@@ -36,10 +36,10 @@
 # * Written by Cedric Bastoul, Cedric.Bastoul@inria.fr                        *
 # *                                                                           *
 # *****************************************************************************/
-if BUNDLED_ISL
-    MAYBE_ISL = isl
-    ISL_LA = $(top_builddir)/isl/libisl.la
-endif
+#if BUNDLED_ISL
+#    MAYBE_ISL = isl
+#    ISL_LA = $(top_builddir)/isl/libisl.la
+#endif
 
 if BUNDLED_OSL
     MAYBE_OSL = osl
